app/views/admin/admin_users/show.html.erb

```diff
@@ -1,110 +1,119 @@
 <div class="admin">
   <!--Descriptive page name, messages and instructions-->
-  <h2 class="heading"><%= ts("User") %>: <%= @user.login %> (<%= @user.id %>)</h2>
+  <h2 class="heading"><%= t(".page_heading", login: @user.login, id: @user.id) %></h2>
   <!--/descriptions-->
 
   <!--subnav-->
-  <p class="note">
-    <%= ts("To fix common errors with this user's Subscriptions and Stats pages, and to reindex their works and bookmarks, choose \"Troubleshoot Account.\"") %></p>
   <ul class="navigation actions" role="navigation">
     <li>
-      <%= link_to ts("Manage User Invitations"), manage_user_invitations_path(user_id: @user.login) %>
+      <%= link_to t(".navigation.invitations.manage"), manage_user_invitations_path(user_id: @user.login) %>
     </li>
     <li>
-      <%= link_to ts("Add User Invitations"), user_invitations_path(user_id: @user.login) %>
+      <%= link_to t(".navigation.invitations.add"), user_invitations_path(user_id: @user.login) %>
     </li>
-    <li><%= link_to ts("Troubleshoot Account"), troubleshoot_admin_user_path(id: @user.login) %></li>
+    <li><%= link_to t(".navigation.troubleshoot"), troubleshoot_admin_user_path(id: @user.login) %></li>
     <li>
-      <%= button_to ts("Send Activation Email"), { action: "send_activation", id: @user }, disabled: @user.active? %>
+      <%= button_to t(".navigation.activate"), { action: "activate", id: @user }, disabled: @user.active? %>
     </li>
     <li>
-      <%= button_to ts("Activate User Account"), { action: "activate", id: @user }, disabled: @user.active? %>
+      <%= link_to t(".navigation.roles"), { action: "index", user_id: @user.id } %>
     </li>
     <li>
-      <%= link_to ts("Manage User Roles"), { action: "index", name: @user.login, role: "0" } %>
+      <%= link_to t(".navigation.profile"), user_profile_path(@user) %>
     </li>
   </ul>
   <!--/subnav-->
 
   <!--main content-->
-  <h3 class="heading"><%= ts("Abuse Administration") %></h3>
+  <p class="note"><%= t(".note") %></p>
+  <h3 class="heading"><%= t(".info.heading") %></h3>
+  <div class="wrapper">
+    <dl class="user meta group">
+      <dt><%= t(".info.email") %></dt>
+      <dd><%= @user.email %></dd>
+      <dt><%= t(".info.invitation") %></dt>
+      <dd><%= @user.invitation ? link_to(@user.invitation.id, invitation_path(@user.invitation.id)) : t(".info.no_invitation") %></dd>
+    </dl>
+  </div>
+
+  <h3 class="heading"><%= t(".fnok.heading") %></h3>
   <%= form_tag action: "update_next_of_kin", controller: "admin/admin_users" do %>
     <%= error_messages_for @user.fannish_next_of_kin %>
     <%= hidden_field_tag "user_login", @user.login %>
-    <fieldset<% unless policy(@user).can_manage_next_of_kin? %> disabled="disabled"<% end %>>
-      <legend><%= ts("Fannish Next of Kin") %></legend>
```

Comment:
Do we still need a `<legend>` here, or is it not needed if it's just one field set?

Reply:
I was going back and forth on it myself. It's not required, and we have the heading, so it could be redundant, especially since we usually include a backup heading under the legend.

But I think I'll add legends to both field sets and let the existing headings serve as the backups.